src/app/App.test.tsx

```diff
@@ -15,14 +15,14 @@ test('renders Auth page link', () => {
   expect(linkElement).toBeInTheDocument();
 });
 
-test('Auth page link works', () => {
+test('Auth page link works', async () => {
   render(
     <Provider store={createTestStore()}>
       <App />
     </Provider>
   );
   const linkElement = screen.getByText(/Auth/i);
   linkElement.click();
-  const LoginStatusText = screen.getByText(/kbase_session/);
+  const LoginStatusText = await screen.findByText(/You are not logged in/);
   expect(LoginStatusText).toBeInTheDocument();
 });
```

Comment:
This test was randomly failing before using async, maybe due to the re-render triggered by react-router? Unsure

Reply:
Nice catch!